runtime/doc/treesitter.txt

```diff
@@ -1363,7 +1360,7 @@ LanguageTree:register_cbs({cbs}, {recursive})
       • {cbs}        (`table`) An |nvim_buf_attach()|-like table argument with
                      the following handlers:
                      • `on_bytes` : see |nvim_buf_attach()|, but this will be
-                       called after the parsers callback.
+                       called _after_ the parsers callback.
```

Comment:
Do we want to keep this or remove the `_`?

Reply:
Remove, please. Vimdoc has no (text) markup.

Reply:
If we want emphasis, we should document this (as a "Neovimdoc" addition -- e.g., all uppercase). Then we can add treesitter support for it (e.g., make it bold).

Reply:
There's so few of these. It's probably easier if I just remove these from the source.

Reply:
Agreed.